<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="0" name="WDPA_Test" tests="6" failures="5" timestamp="24 May 2019 20:51:45 GMT" time="264.313" errors="0">
  <testcase name="SetEvironmentLevel" time="0.055" classname="SupportClasses.Environment">
    <skipped/>
  </testcase> <!-- SetEvironmentLevel -->
  <testcase name="WDPA_LTLFreight_Anonymous" time="100.578" classname="WDPA_Application.WDPA_SmokeTest">
    <failure type="java.lang.AssertionError" message="Text still present.  By.id: address.accountStateProvince.field1 - ">
      <![CDATA[java.lang.AssertionError: Text still present.  By.id: address.accountStateProvince.field1 - 
at WDPA_Application.WDPA_SmokeTest.WDPA_LTLFreight_Anonymous(WDPA_SmokeTest.java:146)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- WDPA_LTLFreight_Anonymous -->
  <testcase name="WDPA_LTLFreight" time="188.299" classname="WDPA_Application.WDPA_SmokeTest">
    <failure type="java.lang.AssertionError" message="Text still present.  By.id: freightPickupInfo.readyTime - ">
      <![CDATA[java.lang.AssertionError: Text still present.  By.id: freightPickupInfo.readyTime - 
at WDPA_Application.WDPA_SmokeTest.WDPA_LTLFreight(WDPA_SmokeTest.java:133)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- WDPA_LTLFreight -->
  <testcase name="WDPA_ExpressFreight" time="219.579" classname="WDPA_Application.WDPA_SmokeTest">
    <failure type="java.lang.AssertionError" message="Text still present.  By.id: package.expFreight.closeTime - ">
      <![CDATA[java.lang.AssertionError: Text still present.  By.id: package.expFreight.closeTime - 
at WDPA_Application.WDPA_SmokeTest.WDPA_ExpressFreight(WDPA_SmokeTest.java:120)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- WDPA_ExpressFreight -->
  <testcase name="WDPA_Ground" time="229.601" classname="WDPA_Application.WDPA_SmokeTest">
    <failure type="java.lang.AssertionError" message="The system has experienced an unexpected problem and is unable to complete your request. Please contact FedEx Customer Service to schedule your pickup.">
      <![CDATA[java.lang.AssertionError: The system has experienced an unexpected problem and is unable to complete your request. Please contact FedEx Customer Service to schedule your pickup.
at WDPA_Application.WDPA_SmokeTest.WDPA_Ground(WDPA_SmokeTest.java:93)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- WDPA_Ground -->
  <testcase name="WDPA_Express" time="260.728" classname="WDPA_Application.WDPA_SmokeTest">
    <failure type="java.lang.AssertionError" message="Please correct the error(s) in red.">
      <![CDATA[java.lang.AssertionError: Please correct the error(s) in red.
at WDPA_Application.WDPA_SmokeTest.WDPA_Express(WDPA_SmokeTest.java:106)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- WDPA_Express -->
</testsuite> <!-- WDPA_Test -->
